uploading/groa-user-router.test.js

```diff
@@ -0,0 +1,26 @@
+const db = require("../database/dbConfig.js");
+const request = require("supertest");
+const server = require("../api/server.js");
+
+beforeEach(async function() {
+  await db.raw("TRUNCATE users RESTART IDENTITY CASCADE");
+});
+afterEach(async function() {
+  await db.raw("TRUNCATE user_letterboxd_ratings RESTART IDENTITY CASCADE");
+});
+
+describe("POST /users/:user_id/add-movie-rating", () => {
+  it("adds a new movie rating and gives a success message", async () => {
+    await db.seed.run();
+    const res = await request(server)
+      .post("/api/users/1/add-movie-rating")
+      .send({
+        name: "American Psycho",
+        year: 2000,
+        rating: 3
```

Comment:
Needs an id?

Reply:
Do I need to make an id? Does this table not auto increment?

Reply:
I was thinking user_id, anyway, test passes